internal/pkg/cli/svc_init.go

```diff
@@ -468,6 +472,9 @@ func (o *initSvcOpts) askImage() error {
 }
 
 func (o *initSvcOpts) shouldSkipAsking() (bool, error) {
```

Comment:
should we rename this method to `doesManifestAlreadyExist`?

Reply:
Maybe `manifestExists`? Having `does` in a function name seems a little odd to me 🤔

Reply:
Changed it to `manifestAlreadyExists` - a nice middle ground 🤣